src/icon-button.css.ts

```diff
@@ -417,6 +417,7 @@ export const overlay = style({
     width: '100%',
     height: '100%',
     backgroundColor: 'transparent',
+    borderRadius: '50%',
```

Comment:
I have no idea why the overlay was rounded in most cases, but not in Popover (only in iOS).

This fixes the issue